File: 02-Development/front-end/src/services/api/bills.ts
import http from '@utils/axios'

export interface IncomeRecordParams {
	billsName: String
	billsTags: String
	billsDate: String
	billsNum: String
	billsInfo: String
	userEmail: String
}

export const incomeRecordAPI = (params: IncomeRecordParams) => {
	return http.request({
		params,
		method: 'post',
		url: '/bills/incomeRecord'
	})
}

File: 02-Development/server/src/main/java/com/black/entity/Bills.java
package com.black.entity;

import lombok.AllArgsConstructor;
import lombok.Data;
import lombok.NoArgsConstructor;
import org.springframework.boot.context.properties.ConfigurationProperties;
import org.springframework.context.annotation.PropertySource;
import org.springframework.stereotype.Component;

import java.util.Date;

/**
 * @author Black
 */
@Data
@Component
@NoArgsConstructor
@AllArgsConstructor
@ConfigurationProperties(prefix = "bills")
@PropertySource(value = { "classpath:application.yml" })
public class Bills {
    private Integer billsId;
    private String billsName;
    private String billsTags;
    private String billsDate;
    private String billsNum;
    private String billsInfo;
    private String userEmail;
}
